icons/life-buoy.svg

```diff
@@ -10,10 +10,9 @@
   stroke-linejoin="round"
 >
   <circle cx="12" cy="12" r="10" />
+  <path d="m4.9 4.9 4.3 4.3" />
+  <path d="m14.8 9.2 4.3-4.3" />
+  <path d="m14.8 14.8 4.3 4.3" />
+  <path d="m4.9 19.1 4.3-4.3" />
```

Comment:
I'm afraid we have yet another case of overoptimization. 😕
![image](https://github.com/lucide-icons/lucide/assets/17746067/e7ca1a6f-e27a-456b-a9f1-fb78367a591c)

```suggestion
  <path d="m4.93 4.93 4.24 4.24"/>
  <path d="m14.83 9.17 4.24-4.24"/>
  <path d="m14.83 14.83 4.24 4.24"/>
  <path d="m9.17 14.83-4.24 4.24"/>
```